configuration/cdap-site.xml

```diff
@@ -777,168 +674,292 @@
   </property>
 
   <property>
-    <name>dataset.executor.max.instances</name>
-    <value>${master.service.max.instances}</value>
+    <name>explore.executor.container.memory.mb</name>
+    <value>1024</value>
     <description>
-      Maximum number of dataset executor instances
+      Size of Memory in megabytes for each explore executor instance
     </description>
+    <display-name>Explore Executor YARN Memory</display-name>
+    <value-attributes>
+      <type>int</type>
+      <minimum>0</minimum>
+      <maximum>268435456</maximum>
+      <unit>MB</unit>
+      <increment-step>256</increment-step>
+      <overridable>false</overridable>
+    </value-attributes>
   </property>
 
```

Comment:
missing ``explore.executor.max.instances`` but that seems to be getting deprecated/removed too?